### PR TITLE
[FIVE-401] Crear tabla Resources

### PR DESCRIPTION
**Pull request by gonzalorom**
### Descripción: 
Se debe crear la tabla **Resources** en base de datos con los siguientes atributos.
- Id
- RoleName
- Description
- SalaryPerMonth
- WorkloadCapacity

Además de esto, se debe crear el **entityModel** correspondiente a esta tabla.

### Realizado:
imagen

- Se creó tabla **Resources** y se añade **Script_Create_Resources** en carpeta _DbScripts_
- Se creó entity model Resource.
- Se creó model **Resource**.
- Se agregó DbSet de nueva entity "Resource" en DataAccessContext.

### CODE REVIEW

File: FiveRockingFingers/FRF.Core/Models/Resources.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace FRF.Core.Models
+{
+    public class Resources
+    {
+        public int Id { get; set; }
+        public string RollName { get; set; }
```
Comment (martinmena) [inline FiveRockingFingers/FRF.Core/Models/Resources.cs:6]: _RoleName_

File: FiveRockingFingers/FRF.DataAccess/EntityModels/Resources.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace FRF.DataAccess.EntityModels
+{
+    public class Resources
+    {
+        public int Id { get; set; }
+        public string RollName { get; set; }
```
Comment (martinmena) [inline FiveRockingFingers/FRF.DataAccess/EntityModels/Resources.cs:6]: _RoleName_

**Review by martinmena — APPROVED** [review]

File: FiveRockingFingers/FRF.Core/Models/Resources.cs (hunk 1 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Core/Models/Resources.cs]: Done.

File: FiveRockingFingers/FRF.DataAccess/EntityModels/Resources.cs (hunk 2 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.DataAccess/EntityModels/Resources.cs]: Done.

**Review by gperelstein — APPROVED** [review]

File: FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE [dbo].[Resources](
+	[Id] [int] IDENTITY(1,1) NOT NULL,
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql:2]: What is `IDENTITY(1, 1)`?

File: FiveRockingFingers/FRF.Core/Models/Resources.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace FRF.Core.Models
+{
+    public class Resources
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.Core/Models/Resources.cs:3]: This one should be singular, right?

**Review by matiasbeckerle — APPROVED** [review]
I don't know the business requirements but from technical perspective this looks good.

File: FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql (hunk 3 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql:2]: This means that SQL will set an ID and increment it automatic each time that you create/save a new Resource.

Comment (matiasbeckerle) [reply FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql:2]: Ah this is the autoincrement? Lol didn't remembered it was defined this way. Thanks!

Comment (gonzalorom) [reply FiveRockingFingers/FRF.DataAccess/DbScripts/Script_Create_Resources.sql:2]: Haha yes that's the autoincrement.

**Pull request merged**